firmware/shared/controls/DI_driverTorqueRequest.h

```diff
@@ -1,25 +1,14 @@
 #pragma once
 #include <cmath>
-#include <iostream>
-
 namespace ctrl {
 
 // Templating means this function will work on any types that support addition.
-template <typename T, typename B>
-T DI_driverTorqueRequest(T READY_TO_DRIVE, T DI_Error, B AccelPedPos) {
-    T u1 = 0.0;
-    if ((!READY_TO_DRIVE) == 1 || DI_Error == 1) {
-        return u1;
+template <typename B>
+B DI_driverTorqueRequest(bool READY_TO_DRIVE, bool DI_Error, B AccelPedPos) {
```

Comment:
All parameters should be spelled with `lower_snake_case` to follow our naming conventions. See the [C++ Style Guide -> Variable Names](https://google.github.io/styleguide/cppguide.html#Variable_Names)

Is there a more descriptive name for the error? Where does it come from in the Simulink model / what error does it represent?